### PR TITLE
Feature/external configuration

### PR DESCRIPTION
**Pull request by RamonGilabert**

### CODE REVIEW

File: Source/PickerConfiguration.swift [hunk 1]
```diff
@@ -1,6 +1,13 @@
 import UIKit
 
 public class PickerConfiguration {
```
Comment (vadymmarkov) [inline Source/PickerConfiguration.swift]: I guess it could be just `Configuration` with var `imagePicker`?

Comment (RamonGilabert) [inline Source/PickerConfiguration.swift]: Right, but I don't want to conflict with Lightbox and stuff like that, what do you think?

Comment (vadymmarkov) [inline Source/PickerConfiguration.swift]: But you can use it like `ImagePicker.Configuration`, with module name, and also - do we have something else except `ImagePickerConfiguration`, I mean why do wee need 2 classes?

Comment (RamonGilabert) [inline Source/PickerConfiguration.swift]: Yes just thought about it, changing it!

**Comment by RamonGilabert** [comment]
@vadymmarkov done I guess? :smile:

**Comment by zenangst** [comment]
community-gifs-community-21080243-500-194

**Pull request merged**